README.md

```diff
@@ -103,15 +103,15 @@ We use [Next.js](https://nextjs.org/) as our react framework. This handles build
 
 ## Contributing
 
-There are three protected branches in this repository: `dev`, `qa`, and `production`.
+There are three protected branches in this repository: `main`, `qa`, and `production`.
 
-- `production` is the most current code, used in production by The Palace Project. Any time a PR is merged in, a release is tagged and created. A Docker Image is then built by Github Actions, which is finally pushed to Docker Hub with the tag `latest` as well as the specific version tag of the release (ie `2.3.6`). Generally, we only make PRs to `production` from `qa` unless a hotfix is necessary.
-- `qa` is for staging release candidates when they are ready to be QA'd by the community. These will be pushed to docker with the tag `qa` as well as `qa.short_sha` where `short_sha` is from the most recent commit.
-- `dev` is the default branch. This is where PRs with development work should be made. Once determined it is ready for QA, we will make a PR to `qa`, then a subsequent PR to `production` if it is ready.
+- `production` is the most current code, used in production by The Palace Project. Any time a PR is merged in, a release is tagged and created. A Docker Image is then built by Github Actions, which is finally pushed to GitHub Container Registry with a major.minor version tag (e.g. `2.3`), a specific version tag (e.g. `2.3.6`), a tag with the short commit id (e.g. `sha-c5cda3a`), and the tag `production`. Generally, we only make PRs to `production` from `qa` unless a hotfix is necessary.
+- `qa` is for staging release candidates when they are ready to be QA'd by the community. These will be pushed to GitHub Container Registry with a major.minor version tag (e.g. `2.3`), a specific version tag (e.g. `2.3.6`), a tag with the short commit id (e.g. `sha-c5cda3a`), and the tag `qa`.
+- `main` is the default branch. This is where PRs with development work should be made. Once determined it is ready for QA, we will make a PR to `qa`, then a subsequent PR to `production` if it is ready.
```

Comment:
Do we want to maybe simplify this workflow a bit? So its more in line with [ADR-001](https://github.com/ThePalaceProject/architectural-decisions/blob/main/ADR-001%20Github%20Workflow.md)? I think we could probably just have a main branch, and tag releases off that main branch when we want to. I'm not sure if we need the `production` and `qa` branches as well do we?

Reply:
I think this branch structure represents NYPLs dev -> qa -> production workflow, with automated centralized deployments for qa and production. We don't currently do that, so I don't think we need the extra branches here.

Reply:
Ok. Will update this part of the readme.